exotica_python/src/pyexotica.cpp

```diff
@@ -941,7 +943,8 @@ PYBIND11_MODULE(_pyexotica, module)
         .def("get_state_cost", &DynamicTimeIndexedShootingProblem::GetStateCost)
         .def("get_state_cost_jacobian", &DynamicTimeIndexedShootingProblem::GetStateCostJacobian)
         .def("get_control_cost", &DynamicTimeIndexedShootingProblem::GetControlCost)
-        .def("get_control_cost_jacobian", &DynamicTimeIndexedShootingProblem::GetControlCostJacobian);
+        .def("get_control_cost_jacobian", &DynamicTimeIndexedShootingProblem::GetControlCostJacobian)
+        .def("reset_cost_evolution", &DynamicTimeIndexedShootingProblem::ResetCostEvolution);
```

Comment:
Why would you like to have this exposed? And if so, do it on the PlanningProblem, not the specialised one.

Reply:
The idea here was to reuse the same problem in experiments without having to call Set_U and Set_X. I'll get rid of it.

Reply:
Your solver should call `reset_cost_evolution` at the beginning of solve. The user should not have to take care of that.